addons/web/static/src/search/group_by_menu/group_by_menu.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="web.GroupByMenu" owl="1">
-        <Dropdown class="o_group_by_menu btn-group"
+        <Dropdown class="'o_group_by_menu btn-group' + props.class"
```

Comment:
- This will pipe the class to btn-group, making it `btn-group${class}` . Don't think it's the goal ^^
- The props is defined as optional, yet we always pipe it

Reply:
ok for point 1.
not ok for point 2 ^^ (see graph_view.xml)